src/Hit/Issue.hs

```diff
@@ -93,6 +95,14 @@ getIssueTitle num = fetchIssue num >>= \case
     Left err -> errorMessage (show err) >> exitFailure
     Right Issue{..} -> pure issueTitle
 
+assignToIssue :: Id Issue -> IO (Either Error Issue)
+assignToIssue iNum = withOwnerRepo action
+  where action Nothing _ _ = pure $ Left $ UserError "GITHUB_TOKEN lookup failed"
+        action (Just t) o r = do
+          let user = makeName . untagName $ o
+              nameToAssign = Just $ user
+          editIssue t o r iNum (editOfIssue {editIssueAssignee = nameToAssign})
```

Comment:
Hmm, I guess what you wrote means that it will put only the user to the assignee list. But this is not what should be done as you can remove information about other assignees. You should add the user to the list of the assignees instead. Do I get it right?

Reply:
This is indeed the case. The docs for the library we are using are [https://hackage.haskell.org/package/github-0.20/docs/GitHub-Endpoints-Issues.html](here), and they point to these [https://developer.github.com/v3/issues/#edit-an-issue](GitHub API docs). 

I agree that what we want is to *add* the current user to the list of assigned users. Given that the GitHub API does not allow me to "add to assignees", the way that I would go about this is to change the function `assignToIssue` to get the current list of assignees for the issue, extract the [list of issue assignees](https://hackage.haskell.org/package/github-0.20/docs/GitHub-Data-Issues.html#t:Issue), convert the list of [SimpleUser](https://hackage.haskell.org/package/github-0.20/docs/GitHub-Data-Definitions.html#t:SimpleUser) to a `Name User`, add the current user to that list, and then use the new list as the list to assign.

Is this correct? If so, it would require an update from `github-0.20` to `github-0.21`.

Reply:
@vrom911 I'm trying to upgrade from `github-0.20` to `github-0.21` so that we can use `editAssignees`. I do not understand how `github-0.20` is available to use in the project. It looks like we are using [this resolver](https://www.stackage.org/nightly-2019-02-24), but that resolver does not have the `github` package, and that package is not included in our `stack.yaml`.

Reply:
Regarding `github` library question, it's perfectly fine to upgrade! Especially if it makes life much easier :slightly_smiling_face:  For Stack, we are actually using this nightly: https://www.stackage.org/nightly-2019-02-04 .
But feel free to upgrade to the newest one which is https://www.stackage.org/nightly-2019-02-27 as I can see. And you can add `github` to the extra deps :)

Reply:
Updated the github library, but not the resolver. (I'm trying to touch as little as possible). I also changed the code so that we add to the list of assignees of an issue, rather than over-write.